src/kixi/paloma/enrich/file.clj

```diff
@@ -0,0 +1,63 @@
+(ns kixi.paloma.enrich.file
+  (:require [clojure.data.csv :as csv]
+            [clojure.java.io :as io]))
+
+(defn remove-bom [^String line]
+  (let [bom "\uFEFF"]
+    (if (.startsWith line bom)
+      (.substring line 1)
+      line)))
+
+;; Initial field names, will get revised.
+
+(defn export-business-records->csv [out xs]
+  (csv/write-csv out [["UPRN"
```

Comment:
I've started doing `(into [["UPRN",,,]] (map rowfunc xs))` to create the data structure I want so that I can send it to csv, excel or sql.

Reply:
I've made a note to introduce this when the function is worked on after the merge functions are done.